src/core/public/legacy/legacy_service.ts

```diff
@@ -64,8 +64,8 @@ export class LegacyPlatformService {
 
   public setup({ core, plugins }: SetupDeps) {
     // Always register legacy apps, even if not in legacy mode.
-    core.injectedMetadata.getLegacyMetadata().nav.forEach((navLink: any) =>
-      core.application.registerLegacyApp({
+    core.injectedMetadata.getLegacyMetadata().nav.forEach((navLink: any) => {
+      return core.application.registerLegacyApp({
```

Comment:
NIT: why add the closure brackets/return here?

Reply:
Came about while refactoring this way and that way... Removing!